trailing ;

Compiling with GCC 10 and -pedantic option raises an issue here about a trailing ";"

CGAL-5.2.2/include/CGAL/Mesh_triangulation_3.h:157:3: error: extra ';' [-Werror=pedantic]
  157 |   ;

https://github.com/CGAL/cgal/blob/f10214dadd553fe511f0b3853edd7d0ed5e1ea5b/Mesh_3/include/CGAL/Mesh_triangulation_3.h#L157

Seems easy to remove :-)

--- a/Mesh_3/include/CGAL/Mesh_triangulation_3.h
+++ b/Mesh_3/include/CGAL/Mesh_triangulation_3.h
@@ -154,7 +154,6 @@ private:
                                                      Concurrency_tag> {};
   using Triangulation =
       Mesh_3_regular_triangulation_3_wrapper<Geom_traits, Tds>;
-  ;
 
 public:
   using type = Triangulation;

